src/book/02-system/02-subsystems/03-localisation.mdx

```diff
@@ -5,3 +5,62 @@ title: Localisation
 description: Localisation and how it works in the NUbots codebase.
 slug: /system/subsystems/localisation
 ---
+
+Localisation is the process of estimating the position and orientation of the robot relative to the soccer field. The NUbots codebase uses a Particle Filter approach to estimate the robot's position and orientation relative to the soccer field.
+
+<Grid columns='1fr  1fr' rows="1fr">
+
+![Webots](./images/webots.png 'Webots')
+![NUsight](./images/nusight.png 'Localisation estimate')
+
+</Grid>
+
+### Spaces/Reference Frames
+
+The NUbots codebase uses a number of different reference frames (spaces) for different purposes. The following is a list of the commonly used spaces in the codebase.
+
+| Space    | Letter | Description                                                                                            |
+| -------- | ------ | ------------------------------------------------------------------------------------------------------ |
+| `camera` | C      | Located at camera (left/right) with x pointed "outwards", y to the "left" and z "up"                   |
+| `world`  | W      | Located on the ground below the robot on startup. (Fixed inertial frame)                               |
+| `torso`  | T      | Fixed midway between the robots hip yaw joints, with x "forward", y to the "left" and z "up"           |
+| `robot`  | R      | Located directly beneath torso projected onto the ground, with x "forward", y to the "left" and z "up" |
+| `field`  | F      | Located at the centre circle of the field, with x pointing towards the "own" goal and z "up"           |
```

Comment:
it would be good to keep this kind of information only in one place so we don't have as many places to update the one thing. There is a section on the mathematics page, which I think is still out of date.